src/test/java/it/pagopa/interop/probing/eservice/registry/updater/unit/consumer/ServicesReceiverTest.java

```diff
@@ -44,10 +49,11 @@ void setup() throws IOException {
 		eServiceDTO.setVersionId("226574b8-82a1-4844-9484-55fffc9c15ef");
```

Comment:
Better use a Builder for fluent api programming. It is more readable.

Reply:
done